include/bout/field2d.hxx

```diff
@@ -174,6 +174,9 @@ public:
   /// Return a Region<Ind2D> reference to use to iterate over this field
   const Region<Ind2D>& getRegion(REGION region) const;
   const Region<Ind2D>& getRegion(const std::string& region_name) const;
+  const Region<Ind2D>& getDefaultRegion(const std::string& region_name) const {
```

Comment:
Bikeshedding perhaps, but I think the name `getDefaultRegion` could be confusing: It's getting the field region, using the given region name as the default, rather than getting the default region. Perhaps `getRegionWithDefault`?

Reply:
Since `getRegion` gets the region with the given name or ID, perhaps `getValidRegionWithDefault` is more explicit that it's the valid region that's being requested. Too long?

Reply:
I changed it to `getValidRegionWithDefault`  - that is probably easier to understand. The length seems ok to me :+1: